x/liquidity/client/rest/query.go

```diff
@@ -23,7 +23,7 @@ func queryLiquidityHandlerFn(cliCtx client.Context) http.HandlerFunc {
 		vars := mux.Vars(r)
 		strPoolId := vars[RestPoolId]
 
-		poolID, ok := rest.ParseUint64OrReturnBadRequest(w, strPoolId)
+		poolId, ok := rest.ParseUint64OrReturnBadRequest(w, strPoolId)
```

Comment:
poolID I think is what the linter normally suggest.

Reply:
Thank you for your opinion. I was originally using it as a poolID, but the snake_case of pool_id was automatically converted into a camel case of poolId, so I couldn't find a way to try it, so I'm using it as a temporary measure. In also latest version of cosmos-sdk, [proposal_id](https://github.com/cosmos/cosmos-sdk/blob/d55c1a26657a0af937fa2273b38dcfa1bb3cff9f/proto/cosmos/gov/v1beta1/gov.proto#L49) has been automatically converted to [ProposalId](https://github.com/cosmos/cosmos-sdk/blob/d55c1a26657a0af937fa2273b38dcfa1bb3cff9f/x/gov/types/gov.pb.go#L166).